five.js

```diff
@@ -166,6 +166,9 @@
   }
 
   five.async = function(callback) {
+    if (!callback && typeof Promise !== 'undefined') {
+      return Promise.resolve(five());
```

Comment:
This is not technically asynchronous. Yes the _resolution_ of the promise is but the value fetching is not. This is slightly different then the callback block below that waits one tick before actually calling `five()`. A suggested alternative:

```js
return new Promise(function(resolve) {
  five.async(resolve);
});
```